### PR TITLE
Move shared tests to shared

### PR DESCRIPTION
**Pull request by GantMan**
Shared tests migrated to shared, to help test Net 4.6

### CODE REVIEW

**Comment by rozele** [comment]
I think @matthargett mentioned there would either be NMock support for UWP or a decision to remove it throughout by EOW (which is EOD today).  Can we wait on migrating further tests until that is resolved?

**Comment by rozele** [comment]
None of these tests use NMock, but I don't want to sign off on removing yet more test coverage from UWP

---
In reply to: [257005127](https://github.com/ReactWindows/react-native-windows/pull/837#issuecomment-257005127) [](ancestors = 257005127)

**Comment by GantMan** [comment]
Yup!  Sorry, this existed, and wanted to put it in a stable place for a hand-off if it goes forward.

**Comment by GantMan** [comment]
@rozele - Sounds like there is a plan from @matthargett - so this PR is just for reference as they implement separate testing.

**Comment by matthargett** [comment]
yes. @pre10der89 submitted PR #840 , which moves the few NMock-based tests to the ReactNative.Net46.Tests project, and adds ReactNative.Tests.Shared reference to the ReactNative.Tests Universal project. Once #840 is merged, this PR can be merged at no cost to coverage of the ReactNative Universal project.

**Comment by kevinvangelder** [comment]
I ran into trouble migrating `Bridge/ReactInstanceTests.cs` and `Modules/Storage/AsyncStorageModuleTests.cs` (they ran forever) so I'm not including them in this PR.

**Comment by kevinvangelder** [comment]
So, I guess I can't git... In the previous commit I moved the tests from UWP to shared and then tweaked them to be shared, but I guess that because the end result was closer to the Net46 tests (which I deleted) GitHub has decided to be "helpful" and change it so that I "moved" the Net46 tests and "deleted" the UWP tests...

WAT. THE. WAT.

**Comment by kevinvangelder** [comment]
[Git Status prior to commit](https://api.monosnap.com/rpc/file/download?id=15lvkuCB4wZup2UfFF3IVI7WSzRX2S)

File: ReactWindows/ReactNative.Shared.Tests/Modules/Network/NetworkingModuleTests.cs [hunk 1]
```diff
@@ -10,36 +10,42 @@
 using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
+#if WINDOWS_UWP
 using Windows.Web.Http;
+#else
+using System.Net;
+using System.Net.Http;
+using HttpStreamContent = System.Net.Http.StreamContent;
+#endif
```
Comment (rozele) [inline ReactWindows/ReactNative.Shared.Tests/Modules/Network/NetworkingModuleTests.cs]: I don't think this is needed, we were only using it when HttpStreamContent was an output of a method.  Here, you're just making constructor calls isolated in #ifs anyway, so its better not to type alias.

**Comment by kevinvangelder** [comment]
Replaced by #889 & #899

**Comment by rozele** [comment]
@kevinvangelder can we close this if its superceded by #889 and #899?

**Comment by kevinvangelder** [comment]
@rozele Yes please, I didn't open the PR so I couldn't close it and hadn't gotten around to asking @GantMan to close it yet.

**Pull request closed without merging**